src/pages/tickets/org/orgHeader/index.tsx

```diff
@@ -431,19 +471,20 @@ export const OrgHeader = ({
   const closeStatusPopover = () => setIsStatusPopoverOpen(false);
 
   const selectedWidget = 'wanted';
+
   const filterRef = useRef<HTMLDivElement | null>(null);
-  const { website, github } = organizationUrls;
+
   const handlePostBountyClick = () => {
     setIsPostBountyModalOpen(true);
   };
   const handlePostBountyClose = () => {
     setIsPostBountyModalOpen(false);
   };
-  const handleWebsiteButton = (websiteUrl: string) => {
+  const handleWebsiteButton = (websiteUrl?: string) => {
     window.open(websiteUrl, '_blank');
   };
 
-  const handleGithubButton = (githubUrl: string) => {
+  const handleGithubButton = (githubUrl?: string) => {
```

Comment:
why are we making these optional? shouldn't they always require a string otherwise the page would open an empty page

Reply:
The website url parameter returns undefined if the url is not present, which at the moment  is in most case